src/contracts/Repository/SearchService.php

```diff
@@ -120,11 +137,11 @@ interface SearchService
      * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException if query is not valid
      *
      * @param \Ibexa\Contracts\Core\Repository\Values\Content\Query $query
-     * @param array $languageFilter Configuration for specifying prioritized languages query will be performed on.
+     * @param array<string, mixed> $languageFilter Configuration for specifying prioritized languages query will be performed on.
```

Comment:
```suggestion
     * @param array<int, string>|array<string, mixed> $languageFilter Configuration for specifying prioritized languages query will be performed on.
```
See [`\Ibexa\Core\Search\Legacy\Content\Handler::findContent` magic at beginning](https://github.com/ibexa/core/blob/v4.6.16/src/lib/Search/Legacy/Content/Handler.php#L132-L138)